src/utility/discord.utility.ts

```diff
@@ -0,0 +1,79 @@
+import {
+    REST,
+    type Guild,
+    Routes,
+    ChannelType,
+    type TextChannel,
+    type Interaction
+} from 'discord.js';
+import Commands from '../commands';
+import { NmSecrets } from '../utility';
+
+import {} from 'discord.js';
+import { GuildServiceModel } from '../model';
+
+export function GetChannelByName(guild: Guild | null, name: string) {
```

Comment:
function name are usually lower case in js

Reply:
Except I think not always so when they are first level exports.

Reply:
Only constructors and types should be initialized according to what I find online. I can't think of any functions in core js, node, or any other functions from big libs I've used that are capitalized. But if you would prefer do them upper case, we can do it that way,

Reply:
So I think where this is coming from for me is that if we were creating a module then this would be the case. I'm used to using static methods for utilities, but we've had a discussion and you would prefer to have standalone utilities. I'm fine with this because it's simple. But I would like a way to be clear at a glance if I'm using something that's a common utility export function. The convention for lower case functions locally is established. It makes sense to me to capitalize functions that are exported like this. I think this is how React components are exported even functional ones. So if it's all the same I'd prefer to keep the capitals for now.

Reply:
That's cause react component function are considered constructors, but either way sounds good. I can do upper case exports like that